src/Bicep.Cli/Arguments/BuildParamsArguments.cs

```diff
@@ -80,14 +64,9 @@ public BuildParamsArguments(string[] args) : base(Constants.Command.BuildParams)
                 throw new CommandLineException($"The parameters file path was not specified");
             }
 
-            if (OutputParamsFile is not null && OutputBicepFile is not null && OutputParamsFile == OutputBicepFile)
-            {
-                throw new CommandLineException($"The path for --outfile-params and --outfile-bicep can not be the same");
-            }
-
-            if ((OutputParamsFile is not null || OutputBicepFile is not null) && OutputToStdOut)
+            if ((OutputFile is not null) && OutputToStdOut)
```

Comment:
[nit]: You don't need to wrap `OutputFile is not null` in parenthesis. [`is` has a higher precedence than `&&`](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/#operator-precedence)